packages/legend-query-builder/src/stores/shared/ValueSpecificationEditorHelper.ts

```diff
@@ -293,21 +301,51 @@ export const generateVariableExpressionMockValue = (
 
 export const getValueSpecificationStringValue = (
   valueSpecification: ValueSpecification,
+  applicationStore: ApplicationStore<
+    LegendApplicationConfig,
+    LegendApplicationPluginManager<LegendApplicationPlugin>
+  >,
+  omitEnumOwnerName?: boolean,
```

Comment:
do you think if it's better to change this input to be something like 
```
option: {
omitEnumOwnerName?: boolean,
} 
```

Reply:
Good idea! I've made that change.